backend/fms_core/coordinates.py

```diff
@@ -77,13 +77,14 @@ def validate_and_normalize_coordinates(coords: str, spec: CoordinateSpec) -> str
 
     return c
 
-
-def check_coordinate_overlap(queryset, obj, parent, obj_type: str = "container"):
+def detect_coordinate_overlap(queryset, obj, parent, obj_type: str = "container"):
     """
     Check for coordinate overlap with existing child containers/samples of the
     parent using a queryset, assuming that the queried model has a coordinates
     field which specifies possibly-overlapping item locations.
     """
-    existing = queryset.exclude(pk=obj.pk).get(coordinates=obj.coordinates)
-    raise CoordinateError(f"Parent container {parent} already contains {obj_type} {existing} at "
-                          f"coordinates {obj.coordinates}")
+    error = None
+    exists = queryset.exclude(pk=obj.pk).filter(coordinates=obj.coordinates).exists()
+    if exists:
+        error = f"Parent container {parent} already contains {obj_type} at coordinates {obj.coordinates}"
```

Comment:
You removed the object identifier from the error message. You would have to make a second query here to get it.

Reply:
This change was reverted